src/nxdoc/nuxeo-server/installation/docker-image/build-a-custom-docker-image.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Build a Custom Docker Image
+review:
+  date: '2021-01-21'
+  status: ok
+labels:
+  - multiexcerpt
+  - multiexcerpt-include
+toc: true
+description: Discover how to build a custom Docker image.
+tree_item_index: 100
+---
+
+To build an application from Nuxeo, most of the time, you will need to customize the Nuxeo Docker Image. Let's see how you can do this.
+
+## Installing Nuxeo Packages
+
+We provide a utility script to install remote Nuxeo packages from [Nuxeo Connect](https://connect.nuxeo.com/) and local Nuxeo packages when building an image from the Nuxeo image:
+
+For instance, you can use this script in the following `Dockerfile`:
+
+```Dockerfile
+FROM docker.packages.nuxeo.com/nuxeo/nuxeo
+
+ARG CLID
+ARG CONNECT_URL
+
+COPY --chown=900:0 path/to/local-package-nodeps-*.zip $NUXEO_HOME/local-packages/local-package-nodeps.zip
+COPY --chown=900:0 path/to/local-package-*.zip $NUXEO_HOME/local-packages/local-package.zip
+
+# Install a local package without its dependencies (`mp-install --nodeps`)
+RUN /install-packages.sh --offline $NUXEO_HOME/local-packages/local-package-nodeps.zip
+# Install remote packages and a local package with its dependencies
+RUN /install-packages.sh --clid ${CLID} --connect-url ${CONNECT_URL} nuxeo-web-ui nuxeo-drive $NUXEO_HOME/local-packages/local-package.zip
+RUN rm -rf $NUXEO_HOME/local-packages
+```
+
+## Adding Configuration Properties
+
+As described in this [section]({{page page='configuration-parameters-index-nuxeoconf'}}#docker-image) about the `nuxeo.conf` file, the files from the `/etc/nuxeo/conf.d` directory of the Nuxeo image have their properties appended to the final `nuxeo.conf` file used by the server.
+
+If you want to embed some additional configuration properties in your custom image, you can simply copy some properties files to the `/etc/nuxeo/conf.d` directory when building the custom image.
+
+For instance:
+
+```Dockerfile
+FROM docker.packages.nuxeo.com/nuxeo/nuxeo
+
+COPY /path/to/my-configuration.properties /etc/nuxeo/conf.d/my-configuration.properties
```

Comment:
Don't we need `--chown=900:0 `?

Reply:
Not that I've tested, somehow it's working like this.